documentation/source/user_section/tutorials/shape-analysis/normalizing-morphometrics-compressions/generating-necessary-inputs.rst

```diff
@@ -0,0 +1,84 @@
+.. _generating-necessary-inputs:
+
+Generate the necessary input files
+#############################################################
+
+The ``sct_compute_compression`` command uses the shape metric of the spinal cord at the level of compression and computes a ratio with non-compressed levels above and below the compression site.
+
+
+To use ``sct_compute_compression``, we need the 3 following input files:
+
+
+1. Spinal cord segmentation
+2. Vertebral labels
+3. Compression labels (``t2_compressed_label-compression.nii.gz``)
+
+You can also get these files by downloading :sct_tutorial_data:`data_normalizing-morphometrics-compression.zip`.
+
+1. Spinal cord segmentation
+----------------------------
+.. code:: sh
+
+   sct_deepseg_sc -i t2_compressed.nii.gz -c t2 -qc ~/qc_singleSubj
+
+:Input arguments:
+   - ``-i`` : Input image.
+   - ``-c`` : Contrast of the input image.
+   - ``-qc`` : Directory for Quality Control reporting.
+
+
+:Output files/folders:
+   - ``t2_compressed_seg.nii.gz`` : Spinal cord segmentation.
+
+
+2. Label the spinal cord vertebral levels
+------------------------------------------
+.. code:: sh
+
+   sct_label_vertebrae -i t2_compressed.nii.gz -s t2_compressed_seg.nii.gz -c t2 -qc ~/qc_singleSubj
+
+:Input arguments:
+   - ``-i`` : The input image file.
+   - ``-s`` : Spinal cord segmentation.
+   - ``-c`` : Contrast of the input image.
+   - ``-qc`` : Directory for Quality Control reporting.
+
+:Output files/folders:
+   - ``t2_compressed_seg_labeled.nii.gz`` : Spinal cord vertebral labels.
+
+3. Generate spinal cord compression labels
+-------------------------------------------
+
+1. Open the image in FSLeyes. If the image does not have sufficient axial and sagittal resolution, open two individual images (axial image and sagittal image) in FSLeyes.
+
+.. code:: sh
+
+   fsleyes t2_compressed.nii.gz &
+
+2. Look at the clinical data provided with compression information to know at which level the compression(s) are located.
+3. Locate the compression in the sagittal view.
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/sb/4158-add-tutorial-sct-compute-compression/spinalcord-compresssion-norm/localizing_compression_sag.png
+   :align: center
+
+4. In the axial view, toggle to the maximum compressed slice around the compression.
```

Comment:
I got confused at this step, since there are no pictures.

I wasn't sure what the "maximum compressed slice" was supposed to be. Is it the LR slice, the AP slice, or the IS slice? In which direction should I be toggling?

Reply:
> (I got a bit confused at this step, since there are no pictures.)

Do you think that a short video tutorial would be better than static screenshots?

Reply:
since we specify in the axial view, it is the axial slice (IS slice)

Reply:
> since we specify in the axial view, it is the axial slice (IS slice)

Ahhhh okay, I get it now!! So, this means "In the axial view, increase/decrease the last "voxel location" value to view the axial slices above and below the compression site. This will allow you to determine which axial slice has the greatest compression."

![image](https://github.com/spinalcordtoolbox/spinalcordtoolbox/assets/16181459/c41a7063-9a57-46dd-abeb-8027dbd6ac37)

Reply:
> Do you think that a short video tutorial would be better than static screenshots?

I don't think a video is 100% necessary? I think I just needed the wording clarified to me. :)

Reply:
yes!